docs/pages/distribution/runtime-version.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Runtime Version
+---
+
+Expo Update allows you to publish over the air updates to apps. The update must be for an app running a
+[compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. You may use a runtime other than an Expo SDK, by specifying a runtime version.
+
+The runtime version should be specified in your `app.json`:
+
+```typescript
+{
+	expo: {
+		...
+		runtimeVersion: "2.718",
+	}
+}
+```
+## Publishing
+
+[Publishes](../workflow/publishing/#how-to-publish) run with the runtime version set in the `app.json` will be delivered to builds running the same runtime version.
+
+## Builds
+
+Apps running a custom runtime version can be built with [EAS Build](../build/introduction/).
+
+There are two ways to set the runtime version of a build.
```

Comment:
these should be split up based on whether you have a managed app or not. if you have a managed app then you should just update your app config (app.json/app.config.js), otherwise you should edit native files directly. we should leave out `expo prebuild`